solver/src/solver/http_solver/model.rs

```diff
@@ -57,20 +58,20 @@ pub struct TokenInfoModel {
 #[derive(Debug, Serialize)]
 pub struct CostModel {
     pub amount: u128,
```

Comment:
This can run into precision problems since a fair share of parsers use IEEE double precision floating point numbers for JSON numbers. This means that for values larger than `2^53`, there will be loss of precision. This should probably use `display_fromstr` formatting.